app/models/masterkey.js

```diff
@@ -2,21 +2,23 @@ const mongoose = require('mongoose');
 const _ = require('lodash');
 
 const masterKeySchema = new mongoose.Schema({
+  type: { type: String, default: 'xpub', enum: ['xpub', 'xlm'] },
   coin: { type: String },
   customerId: { type: String },
-  xpub: { type: String },
+  pub: { type: String },
```

Comment:
Can you talk me through what happens in a production environment with changes like this? What happens if there were already documents in the collection with `xpub` attrs set? Do we need a schema + data migration?

Reply:
Thankfully this project isn't deployed anywhere yet, so we can make these non-backwards compatible convention changes before release :)